experiments/colgranule/vector_graph_persisted_bench_test.go

```
@@ -0,0 +1,594 @@
+		b.Run(fmt.Sprintf("100k/%s/%s", columnVectorGraphPersistedOrderLocal, compression), func(b *testing.B) {
+			var totalBuildNanos int64
+			var totalOpenNanos int64
+			var totalDecodeNanos int64
+			var lastFixture columnVectorGraphPersistedFixture
+			b.ReportAllocs()
+			for i := 0; i < b.N; i++ {
+				fixture := buildColumnVectorGraphPersistedFixture(b, 100_000, 128, 16, 8192, compression, columnVectorGraphPersistedOrderLocal)
+				totalBuildNanos += fixture.buildNanos
+				totalOpenNanos += fixture.openNanos
+				totalDecodeNanos += fixture.decodeNanos
+				lastFixture = *fixture
```

golangci-lint (2.12.2)

[error] copylocks: assignment copies lock value to lastFixture: github.com/snissn/gomap/experiments/colgranule.columnVectorGraphPersistedFixture contains sync.Once contains sync.noCopy

(govet)